index.html

```diff
@@ -162,26 +162,47 @@ <h2>Introduction</h2>
   <p><a class="termref">Accessibility <abbr title="Application Programming Interfaces">APIs</abbr></a> currently support flat, unstructured strings for accessible names and descriptions.  The result of the name/description computation is thus a flat string. </p>
   <p>The terms "accessible name" and "accessible description" are used to emphasize that they are properties of <a class="termref">accessible objects</a> as exposed by <a class="termref">Accessibility <abbr title="Application Programming Interfaces">APIs</abbr></a>.  However, they are frequently referred to hereafter as simply "name" and "description".</p>
 </section>
+<section class="informative" id="terms">
+	<h1>Important Terms</h1>
+	<div>
+		<p>While some terms are defined in place, the following definitions are used throughout this document. </p>
+		<dl class="termlist">
+      <dt><dfn data-export="">Accessible Description</dfn></dt>
+			<dd>
+			  <p>An accessible description provides additional information, related to an interface element, that complements the <a>accessible name</a>. The accessible description might or might not be visually perceivable. </p>
+			</dd>
+			<dt><dfn data-export="">Accessible Name</dfn></dt>
+			<dd>
+			  <p>The accessible name is the name of a user interface element. Each platform <a>accessibility <abbr title="application programming interface">API</abbr></a> provides the accessible name property. The value of the accessible name may be derived from a visible (e.g., the visible text on a button) or invisible (e.g., the text alternative that describes an icon) property of the user interface element.  See related <a>accessible description</a>.</p>
+			  <p>A simple use for the accessible name property may be illustrated by an &quot;OK&quot; button. The text &quot;OK&quot; is the accessible name. When the button receives focus, assistive technologies may concatenate the platform's role description with the accessible name. For example, a screen reader may speak &quot;push-button OK&quot; or &quot;OK button&quot;. The order of concatenation and specifics of the role description (e.g., &quot;button&quot;, &quot;push-button&quot;, &quot;clickable button&quot;) are determined by platform <a class="termref">accessibility API</a>s or <a>assistive technologies</a>.</p>
+			</dd>      
+      <dt><dfn>Tooltip attribute</dfn></dt>
+			<dd>
+			  <p>Any host language attribute that would result in a user agent generating a tooltip such as in response to a mouse hover in desktop user agents.</p>
```

Comment:
What about SVG title element? (not attribute)  Should the definition be broadened to include that? e.g. "Any host language feature that would result in a user agent generating a tooltip..."

Reply:
Tracking this as a separate issue: https://github.com/w3c/accname/issues/151